config/docker/cros-baseline.jinja2

```diff
@@ -20,7 +20,7 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
 {{ super() }}
 
 ADD https://raw.githubusercontent.com/kernelci/kernelci-core/\
-kernelci.org/\
+main/\
```

Comment:
@nuclearcat I'm not sure about the significance of kernelci.org branch. But dmesg.sh file is same on both main and kernelci.org.

Reply:
yes, at current moment better to use main